fiftyone/core/labels.py

```diff
@@ -2103,3 +2188,547 @@ def _from_geo_json(d):
         polygons = None
 
     return points, lines, polygons
+
+
+class SMPLParams(EmbeddedDocument):
```

Comment:
Using `DynamicEmbeddedDocument` instead of `EmbeddedDocument` is usually preferred because it allows users to store additional attributes on these documents when using this class to store data on their samples.

Another reason is backwards/forwards compatibility. Like protocol buffers, we want the flexibility of attributes being added or removed in the future without causing validation errors.